zetasql/fuzzing/common.h

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Common utilities for fuzzers
+
+#ifndef ZETASQL_FUZZING_COMMON_H_
+#define ZETASQL_FUZZING_COMMON_H_
+
+#define CONV_FN(TYPE, VAR_NAME) std::string TYPE##ToString(const TYPE& VAR_NAME)
```

Comment:
Why not just pure C++? It's a nice language.

Reply:
So the convention is assume here: https://chromium.googlesource.com/chromium/src/third_party/+/refs/heads/master/sqlite/fuzz/sql_query_proto_to_string.cc#22

I think using macro does clean up code a little. If we need to cover the complete the entire syntax structure, we'll need **many** #ToString methods, as demonstrated in the code above. Having a macro here minimizes the probability of getting a typo. I do agree however, that CONV_FN may be a bad name for revealing its intention.

Reply:
I also agree with Zhengdai here--I think the macro actually does clean things up slightly.

I do think it'd make sense to move it into the file it's used in (partly just to keep it scoped as close to the usage as possible and prevent it from spreading without us thinking more about it).  I'd also like to see some comments that show the usage & effect. :)